test/web-platform-tests/to-upstream/html/semantics/embedded-content/the-img-element/remove-attribute-src.html

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
+<meta charset="utf-8">
+<title>Disabled checkbox event handling</title>
+<link rel="author" title="atsikov" href="mailto:alexey.tsikov@gmail.com">
+<link rel=help href="https://html.spec.whatwg.org/#the-img-element">
+
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+
+<body>
+<script>
+"use strict";
+
+function testRemoveSrc(image, t) {
+  image.removeAttribute("src");
+  assert_equals(image.width, 0);
+  assert_equals(image.height, 0);
+  assert_equals(image.complete, false);
+  assert_equals(image.src, "");
+  assert_equals(image.currentSrc, "");
+  t.done();
```

Comment:
This will cause the test to finish, even if onload/onerror are going to be filed. I think you'll want to put this inside a `t.step_timeout()` call with a comment like "Wait for a bit to make sure onload/onerror are not fired".

Reply:
`t.done()` was put here intentionally. Same checks as for timeout are performed for initial `onload` handler to make sure loaded image was cleared upon attribute removal. Then `t.done()` should be called for both cases.

Reply:
Actually I decided that since `t.step_timeout` will be removed in case of moving test to wpt repo, there's no need for a separate function. Code is a bit duplicated with timeout handler but then will be more straightforward without additional function.